tests/DemoCallExitTest.php

```diff
@@ -305,4 +318,16 @@ public function JSONResponsePOSTDataProvider()
 
         return $files;
     }
+
+    /**
+     * Set directory with proper separator.
+     *
+     * @param $dir
+     *
+     * @return string
+     */
+    private function getPath(string $dir): string
+    {
+        return DIRECTORY_SEPARATOR . $dir . DIRECTORY_SEPARATOR;
```

Comment:
Is this URL path or local path? If URL, `DIRECTORY_SEPARATOR` is wrong (there should be always `/`), if local/fs, then prefix with `__DIR__`.

Reply:
This is for getResponseFromRequestGET($path), so I guess it is about url. I  will change it for '/'

Reply:
fix